tools/prow-config-updater/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"flag"
+	"log"
+
+	"knative.dev/pkg/test/ghutil"
+)
+
+func main() {
+	mainGithubTokenFile := flag.String("main-github-token-file", "",
```

Comment:
This tool is not intended to be run locally, as running it will require specific Github token files, it does support dryrun option for debugging purpose though.

Reply:
Drop `main-` maybe?

Reply:
Done